src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+import {
+  startGame, getScores, postScore, render,
+} from './modules/Apiconnectivity.js';
+
+const addForm = document.querySelector('form');
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
+const refresh = document.querySelector('button');
+
+let gameId;
+const startGames = () => {
+  startGame('New Game Satrt Session')
+    .then((response) => response.result.split(' '))
+    .then((res) => {
+      [gameId] = [res[3]];
+    });
+};
+
+const getScoress = () => {
+  getScores(gameId)
+    .then((response) => render(response.result));
+};
+
+const postScores = (e) => {
+  e.preventDefault();
+  postScore(gameId, nameInput.value, scoreInput.value);
+  addForm.reset();
+};
```

Comment:
> Use async and await JavaScript features to consume the API.

- [ ] Also looking at the project requirement, you are meant to use the `async await pattern` to make requests instead of the `.then`. I'll suggest you update this code to meet the requirement. :+1: :+1: